dts/arm/silabs/efr32mg.dtsi

```diff
@@ -245,7 +245,7 @@
 				compatible = "silabs,gecko-pwm";
 				status = "disabled";
 				label = "PWM_0";
-				#pwm-cells = <2>;
+				#pwm-cells = <3>;
```

Comment:
The same should be done in efr32fg1p.dtsi: https://github.com/zephyrproject-rtos/zephyr/blob/39ed4b879b4c7b6136f6d094183d870adce8cfda/dts/arm/silabs/efr32fg1p.dtsi#L187

Reply:
done thx